src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -1049,13 +1049,12 @@ private void DynamoViewModelRequestSaveImage(object sender, ImageSaveEventArgs e
 
         private void DynamoViewModelRequestSave3DImage(object sender, ImageSaveEventArgs e)
         {
-            var canvas = (DPFCanvas)BackgroundPreview.View.RenderHost;
-
+            var bitmapSource =BackgroundPreview.View.RenderBitmap();
```

Comment:
Could you explain this change?

Reply:
yep, this crashed on the new helix version, so I moved to the new helix api.